packages/stores/src/queries-external/ibc/status-by-id.ts

```diff
@@ -42,8 +42,8 @@ class ObservableQueryIbcChainStatus extends ObservableQueryExternalBase<
       );
       if (channelData) {
         if (channelData.size_queue > 0) {
-          if (channelData.duration_minutes > 20) return "blocked";
-          else if (channelData.duration_minutes > 5) return "congested";
+          //if (channelData.duration_minutes > 20) return "blocked";
+          /*else */if (channelData.duration_minutes > 5) return "congested";
```

Comment:
Perhasp we can raise the bar with what is considered blocked and what is considered congested? Perhaps if the queue is larger, and the times are larger

Reply:
I think it's less consequential to have https://info.osmosis.zone/ibc have a lower bar